### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by nelsinofrancisco**
# Hello, you! :raised_hands: 

> In this Pr, we're submitting our 2 persons group project, we followed the requirements to build a recipe_app, where the user can add ingredients(which is food), add foods (which can become an ingredient for a recipe), can add a recipe(where the ingredients can be added to). The user can also see public recipes and we make a sum list of all items and prices for buying all the ingredients needed to make all the available recipes for the logged_user!

## Bullet Gem

I think bullet gem it's already enabled, but you can double-check it with: `bundle exec rails g bullet:install`

## Thank you for the Feedback.

### CODE REVIEW

**Comment by oloomoses** [comment]
Check slack dm

**Review by oloomoses — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that Does not have a descriptive readme as per the project requirements.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/recipes/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          <% end %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+<% else %>
+  <div class='recipes-container'>
+    <h1>Unauthorized access, please login to continue!</h1>
+  </div>
+<% end %>
```
Comment (descholar-ceo) [inline app/views/recipes/index.html.erb:24]: - [x] Food index page should look like this wireframe
Screenshot from 2022-01-08 17-02-20

- [x] Colors used make it very difficult to see what's written on the page, hence it makes your page bad user experience, would you work a bit more about that?
Screenshot from 2022-01-08 17-06-49
=> This comment applies to all pages, please address the colors all over this app

File: app/views/recipes/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        Cooking time: <%= @recipe.cooking_time %> hour
+      </p>
+      <p>
+        <%= @recipe.description %>
+      <p>
+    </div>
+    <div class='recipe-detail-buttons'>
+      <% if user_signed_in? %>
+        <%= link_to general_shopping_lists_path  do %>
+          <button type="button" class='detail-button'>Generate shopping list</button>
```
Comment (descholar-ceo) [inline app/views/recipes/show.html.erb:38]: - [ ] On the recipe add ingredient page, the dropdown select field for food is empty whereas I have foods available, I consider this as a bug, would you fix it?
Screenshot from 2022-01-08 17-09-57

- [x] When clicking on the generate shopping list, the following interface should appear
Screenshot from 2022-01-08 17-12-02

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @nelsinofrancisco and @JbirdL86 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/recipes/show.html.erb (hunk 2 above)
Comment (JbirdL86) [reply app/views/recipes/show.html.erb:38]: @descholar-ceo Thank you for your review, there is one detail about this though. We are only a group of 2 persons so we are not using inventories. Please let us know how to proceed, thank you!

**Review by Emmyn5600 — APPROVED** [review]
Hi @nelsinofrancisco and @JbirdL86 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

-  Work on the styling. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**